src/Guide/Api/Methods.hs

```diff
@@ -104,6 +104,23 @@ deleteItem db itemId = uncache db (CacheItem itemId) $ do
 
 -- TODO: move a trait
 
+-- | Create a trait (pro/con).
+createTrait :: DB -> Uid Item -> TraitType -> Text -> Handler (Uid Trait)
+createTrait db itemId traitType text = do
+    traitId <- randomShortUid
+    (_edit, _newTrait) <- case traitType of
+        Con -> dbUpdate db (AddCon itemId traitId text)
+        Pro -> dbUpdate db (AddPro itemId traitId text)
+    invalidateCache' db (CacheItemTraits itemId)
```

Comment:
Let's have a TODO here for `addEdit edit`, so that we won't forget to do it later.